src/connectionManager.js

```diff
@@ -164,11 +164,12 @@ function replaceAll(originalString, strReplace, strWith) {
 }
 
 function normalizeAddress(address) {
-    // attempt to correct bad input
+    // Attempt to correct bad input
     address = address.trim();
 
-    if (address.toLowerCase().indexOf('http') !== 0) {
-        address = `http://${address}`;
+    if (!address.toLowerCase().startsWith('http')) {
+        // Assume HTTPS for security
+        address = `https://${address}`;
```

Comment:
Not sure if a GH issue displaying the quotation marks ('), but this appears to me as a backtick (`). I think you should change that.

Reply:
It's a template literal, so it's meant to be a backtick.